src/applications/vaos/components/VAOSApp/DowntimeMessage.jsx

```diff
@@ -70,6 +70,9 @@ export default function DowntimeMessage({
             'VA online scheduling will be down for maintenance'
           }
           data-testid="downtime-approaching-modal"
+          secondaryButtonText="Dismiss"
+          onSecondaryButtonClick={close}
+          uswds
```

Comment:
Replacing the button element to utilize `<VaModal>`'s `secondaryButtonText` property and `onSecondaryButtonClick` custom event